client/payment-details/dispute-details/evidence-list.tsx

```diff
@@ -0,0 +1,117 @@
+/** @format **/
+
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { __ } from '@wordpress/i18n';
+import apiFetch from '@wordpress/api-fetch';
+import { useDispatch } from '@wordpress/data';
+
+/**
+ * Internal dependencies
+ */
+import type { IssuerEvidence } from 'wcpay/types/disputes';
+import { useFiles } from 'wcpay/data';
+import Loadable from 'wcpay/components/loadable';
+import { NAMESPACE } from 'wcpay/data/constants';
+import { FileContent } from 'wcpay/data/files/types';
+
+interface Props {
+	issuerEvidence: IssuerEvidence | null;
+}
+const TextEvidence: React.FC< {
+	evidence: string;
+} > = ( { evidence } ): JSX.Element => (
+	// eslint-disable-next-line jsx-a11y/anchor-is-valid
+	<a
+		href={ URL.createObjectURL(
+			new Blob( [ evidence ], { type: 'text/plain' } )
+		) }
+		className="dispute-evidence-link"
+		download="evidence.txt"
+	>
+		{ __( 'Evidence.txt', 'woocommerce-payments' ) }
+	</a>
+);
+
+const FileEvidence: React.FC< {
+	fileId: string;
+} > = ( { fileId } ) => {
+	const { file, isLoading } = useFiles( fileId );
+	const { createNotice } = useDispatch( 'core/notices' );
+
+	const onDownload = async ( e: React.MouseEvent< HTMLAnchorElement > ) => {
+		e.preventDefault();
+		if ( ! file || ! file.id ) {
+			return;
+		}
+		try {
+			const downloadRequest = await apiFetch< FileContent >( {
+				path: `${ NAMESPACE }/file/${ encodeURI( file.id ) }/content`,
+				method: 'GET',
+			} );
+
+			const link = document.createElement( 'a' );
+			link.href =
+				'data:application/octect-stream;base64,' +
+				downloadRequest.file_content;
```

Comment:
There doesn't seem to be a straightforward way to create a blob with base64 content. So went this route.